docs/circuit_cutting/tutorials/gate_cutting_to_reduce_circuit_width.ipynb

```diff
@@ -0,0 +1,239 @@
+      "text/plain": [
+       "<Figure size 1032.66x494.978 with 1 Axes>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuit = EfficientSU2(7, entanglement=\"linear\", reps=2).decompose()\n",
+    "circuit.assign_parameters([0.8] * len(circuit.parameters), inplace=True)\n",
+    "\n",
+    "circuit.draw(\"mpl\", scale=0.8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8638fdf1",
+   "metadata": {},
+   "source": [
+    "#### Specify some observables for which we wish to simulate expectation values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f75e8dd1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "observables = PauliList(\n",
+    "    [\"ZZIIIII\", \"IZZIIII\", \"IIZZIII\", \"IIIIXXI\", \"IIIIXXI\", \"IIIIIXX\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "162a5629",
+   "metadata": {},
+   "source": [
+    "#### Decompose the circuit and observables according to a specified qubit partitioning using `partition_problem`\n",
+    "\n",
+    "*Arguments*\n",
+    "- **circuit**: The circuit to separate\n",
+    "- **observables**: The observables to separate\n",
+    "- **partition_labels**: A sequence of labels. Each label corresponds to the `circuit`\n",
+    "qubit in the same index. Qubits sharing a common partition label will be grouped together, and non-local gates spanning more than one partition will be cut.\n",
+    "    \n",
+    "*Outputs*\n",
+    "- **subcircuits**: The subcircuits resulting from decomposing nonlocal gates and separating the input ``circuit``\n",
+    "- **subobservables**: A dictionary mapping a partition label to a list of\n",
+    "    Pauli observables corresponding only to the specified partitions' qubits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "30326299",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subcircuits, subobservables = partition_problem(\n",
+    "    circuit, \"AAABBBB\", observables=observables\n",
+    ")"
```

Comment:
I think it'd be very instruction after this to have three cells containing these outputs:

- `subobservables`
- `subcircuits["A"].draw("mpl")` (once https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/153/files#r1198084353 is changed)
- `subcircuits["B"].draw("mpl")`